src/command/cmd_funcs.c

```diff
@@ -1858,6 +1858,10 @@ _who_room(ProfWin* window, const char* const command, gchar** args)
             mucwin_show_affiliation_list(mucwin, MUC_AFFILIATION_OUTCAST);
             return;
         }
+        if (g_strcmp0(args[0], "none") == 0) {
+            mucwin_show_affiliation_list(mucwin, MUC_AFFILIATION_NONE);
```

Comment:
Was there really such an affiliation? :D 
I thought we need to get all the joined users and then need to remove all the ones that have an affiliation set.

Reply:
Ahh I read your explanation in the issue now. Thanks!

Reply:
That's what I thought as well.